File: 2022/reverse.go
// cerner_2tothe5th_2022
// Reverse a string to print Cerner 2 to the 5th
// try online here -
package main
import "fmt"

func reverse(str string) (result string) {
    for _, v := range str {
        result = string(v) + result
    }
    return
}

func main() {
    strRev := reverse("ht5 eht ot 2 renreC")
    fmt.Println(strRev)
}

File: 2022/reverse.nim
## cerner_2tothe5th_2022
proc reverse(s: string): string =
  for i in countdown(s.high, 0):
    result.add s[i]

let str1 = "ht5ehtot2 renreC"
let str2 = "erutluC gnireenignE"
echo reverse(str1)
echo reverse(str2)
